backends/files.go

```diff
@@ -59,6 +60,12 @@ func NewFiles(authOpts map[string]string, logLevel log.Level) (Files, error) {
 		return files, errors.New("Files backend error: no password path given")
 	}
 
+	if saltEncoding, ok := authOpts["salt_encoding"]; ok {
+		files.SaltEncoding = saltEncoding
+	} else {
+		files.SaltEncoding = "base64"
+	}
+
```

Comment:
You could set `base64` as default when creating the `files` var and then just set it to the encoding if given.
We should also check that the encoding is valid before assigning it.

Reply:
Added to latest commit.